R/cv_varsel.R

```diff
@@ -68,22 +68,45 @@
 #' @inherit varsel details return
 #'
 #' @note If `validate_search` is `FALSE`, the search is not included in the CV
-#'   so that only a single full-data search is run.
+#'   so that only a single full-data search is run. If the number of observations
+#'   is big, the fast PSIS-LOO-CV along the full-data search path is likely
+#'   to be accurate. If the number of observations is small or moderate,
+#'   the fast PSIS-LOO-CV along the full-data search path is likely to have
+#'   optimistic bias in the middle of the search path. This result can be
+#'   used to guide further actions and the optimistic bias can be greatly
+#'   reduced by using `validate_search=TRUE`.
 #'
-#'   For PSIS-LOO CV, \pkg{projpred} calls [loo::psis()] (or, exceptionally,
+#'   PSIS uses Pareto-\eqn{\hat{k}} diagnostic to assess the reliability of PSIS-LOO-CV.
+#'   Whether the Pareto-\eqn{\hat{k}} diagnostics are shown as warnings, is controlled
+#'   with a global option `projpred.warn_psis` (default is FALSE).
+#'   See [loo::loo-glossary] for how to interpret the Pareto-\eqn{\hat{k}} values and
+#'   the warning thresholds. \pkg{projpred} does not support the usually recommended
+#'   moment-matching (see [loo::loo_moment_match()] and [brms::loo_moment_match()]),
+#'   mixture importance sampling (`vignette("loo2-mixis", package="loo")`),
+#'   or `reloo`-ing ([brms::reloo()]). If the reference model PSIS-LOO-CV
+#'   Pareto-\eqn{\hat{k}} values are good, but there are high Pareto-\eqn{\hat{k}}
+#'   values for the projected models, you can try increasing the number of draws used
+#'   for the PSIS-LOO-CV (`ndraws_pred` with  `refit_prj=TRUE`). If increasing the
+#'   number of draws does not help and if the reference model PSIS-LOO-CV
+#'   Pareto-\eqn{\hat{k}} values are high, and the PSIS-LOO-CV results change substantially
```

Comment:
Here, I'm wondering whether we want to replace "the PSIS-LOO-CV results" by some like "the reference model's PSIS-LOO-CV results" (just for clarity).

Reply:
updated